### PR TITLE
convert authenticate, accounts

### PR DESCRIPTION
**Pull request by bruce-binary**

### CODE REVIEW

File: src/templates/README.md [hunk 1]
```diff
@@ -74,6 +74,7 @@ Rules
 - Use `it.L('...')` instead of `l("...")` for translations.
 - Use `it.url_for('...')` instead of `request.url_for("...")`.
 - Use `it.website_name` instead of `website_name`.
+- Do NOT use `<Element attributeName={true} />`; just use `<Element attributeName />`.
```
Comment (bruce-binary) [inline src/templates/README.md:77]: @aminroosta documentation for {true} thing

File: src/templates/_common/components/forms.jsx [hunk 2]
```diff
@@ -103,7 +103,7 @@ export const SubmitButton = ({
 }) => {
     const content = <React.Fragment>
         {!no_error &&
-            <p id={msg_id || 'msg_form'} className='error-msg no-margin invisible'></p>
+            <p id={msg_id || 'msg_form'} className={`${is_centered ? 'center-text' : 'gr-row'} error-msg no-margin invisible`}></p>
```
Comment (bruce-binary) [inline src/templates/_common/components/forms.jsx]: added this for that missing `center-text` earlier

Comment (aminroosta) [reply src/templates/_common/components/forms.jsx]: 👍

**Comment by aminroosta** [comment]
@brucebinary Could you fix the indentation of components that use `React.Fragment`?

**Pull request merged**